LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Ability_LW_SpecialistAbilitySet.uc

```diff
@@ -302,6 +377,7 @@ static function X2AbilityTemplate AddHackRewardControlRobot_Permanent()
 	Effect.SetDisplayInfo(ePerkBuff_Penalty, Template.LocFriendlyName, Template.GetMyLongDescription(), "img:///UILibrary_PerkIcons.UIPerk_hack_reward", true,,Template.AbilitySourceName);
 	Effect.bRemoveWhenTargetDies = true;
 	Effect.bUseSourcePlayerState = true;
+	Effect.bPersistThroughTacticalGameEnd=true;
```

Comment:
Is this the entirety of the fix for Master Enemy on evaced units? Even if that's the case, I would actually prefer to have this in a separate pull request, since it's tackling a different issue, albeit one connected with the ability you were working on. I won't insist, since it's a bit of work to separate out now, but if you could do it, I'd appreciate it. Makes it easier to track changes in the commit and issue/PR history.